docs/guides/platform/migrate-to-linode/migrate-from-aws-s3-to-linode-object-storage/index.md

```diff
@@ -0,0 +1,308 @@
+---
+slug: migrate-from-aws-s3-to-linode-object-storage
+title: "How to Migrate From AWS S3 to Linode Object Storage"
+description: "This guide includes steps for how to migrate content from AWS S3 to Linode Object Storage using rclone."
+authors: ["John Dutton"]
+contributors: ["John Dutton"]
+published: 2024-09-23
+keywords: ['migrate','migration','object storage','aws','s3','rclone']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+external_resources:
+- '[Linode Object Storage product documentation](https://techdocs.akamai.com/cloud-computing/docs/object-storage)'
+- '[Linode Object Storage guides & tutorials](/docs/guides/platform/object-storage/)'
```

Comment:
should we add an alias from the old /guides/migrate-to-linode-object-storage/ guide to this new S3 one?